include/swift/AST/DiagnosticsSema.def

```diff
@@ -808,6 +808,13 @@ WARNING(sema_import_current_module_with_file,none,
 ERROR(sema_opening_import,Fatal,
       "opening import file for module %0: %1", (Identifier, StringRef))
 
+REMARK(serialization_skipped_invalid_decl,none,
+       "serialization skipped invalid %kind0",
+       (const Decl *))
+ERROR(serialization_failed,none,
+      "serialization of module '%0' failed due to the errors above",
+      (StringRef))
```

Comment:
You should be able to use `const ModuleDecl*` here similarly to the previous one.